packages/editor-ui/src/components/RunDataJson.vue

```diff
@@ -72,14 +72,20 @@ export default defineComponent({
 		const selectedJsonPath = ref(nonExistingJsonPath);
 		const draggingPath = ref<null | string>(null);
 		const displayMode = ref('json');
+		const jsonDataContainerHeight = ref<number>();
 
 		return {
 			externalHooks,
 			selectedJsonPath,
 			draggingPath,
 			displayMode,
+			jsonDataContainerHeight,
 		};
 	},
+	mounted() {
+		const jsonDataContainer = this.$refs.jsonDataContainer as HTMLDivElement;
+		this.jsonDataContainerHeight = jsonDataContainer.offsetHeight;
```

Comment:
have you considered using  https://vueuse.org/core/useElementSize/#usage ?

this implementation won't work properly if for some reason the user resizes the screen

Reply:
Ahh interesting! Will update it to use it. Thanks for the suggestion.  Did not know about it. As you can see, I'm a newbie in the FE.

Reply:
Agree with @r00gm, either that or with manual listener but we need to make sure it's updated when the element is resized.

Reply:
Will apply [this](https://github.com/n8n-io/n8n/pull/10861) PR is merged, as could not get the reference to the div in the template from the setup method. Probably because we are mixing setup method and options API.

Reply:
You should be able, you just need to return it from the `setup` method but still nice that you did the migration.